test/relations.test.js

```diff
@@ -3790,7 +3790,7 @@ describe('relations', function() {
           return Person.findById(personId);
         })
         .then(function(person) {
-          person.passportItem().toObject().should.eql({name: 'Jim'});
+          person.passportItem().toObject().should.eql({name: 'Mitsos'});
```

Comment:
I'm very familiar with relations; Is this test valid? I believe name should be `Jim`; any clue/direction about this?